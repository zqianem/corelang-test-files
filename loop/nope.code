program
  int i;
begin
  i := 0;
  while i=100begin
    i := i + 10;
  endwhile;
  output i;
end
