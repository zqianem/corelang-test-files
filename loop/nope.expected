program
  int i;
begin
  i:=0;
  while i=100 begin
    i:=i+10;
  endwhile;
  output i;
end
0
